src/apiClient.tsx

```diff
@@ -12,361 +12,379 @@ import {
   short_name: string;
-  description: string;
+  description?: string;
+  url?: string;
 }
 
-export interface OrganizationsReadProcedureResponse {
+export interface OrganizationResponse {
   id: number;
-  full_name: string;
+  full_name?: string;
   short_name: string;
+  description?: string;
+  url?: string;
+  created_at: string;
+}
+
+export interface RequirementResponse {
+  id: number;
+  name: string;
   description: string;
   created_at: string;
 }
 
+export interface VaccineAvailabilityExpandedResponse {
+  id: string;
+  numberAvaliable: number;
+  numberTotal?: number;
+  date: string;
+  vaccine?: number;
+  inputType: InputTypeEnum;
+  tags?: string;
+  created_at: string;
+  location: LocationExpandedResponse;
+}
+
 export interface ValidationError {
   loc: string[];
   msg: string;
   type: string;
 }
 
-export interface GetEntriesApiV1EntriesGetQueryParams {
+export interface ListVaccineAvailabilityApiV1VaccineAvailabilityGetQueryParams {
   postalCode?: string;
 }
 
-export type GetEntriesApiV1EntriesGetProps = Omit<
+export type ListVaccineAvailabilityApiV1VaccineAvailabilityGetProps = Omit<
   GetProps<
-    EntriesResponse[],
+    VaccineAvailabilityExpandedResponse[],
     HTTPValidationError,
-    GetEntriesApiV1EntriesGetQueryParams,
+    ListVaccineAvailabilityApiV1VaccineAvailabilityGetQueryParams,
     void
   >,
   "path"
 >;
 
 /**
- * Get Entries
+ * List Vaccine Availability
  */
-export const GetEntriesApiV1EntriesGet = (
-  props: GetEntriesApiV1EntriesGetProps,
+export const ListVaccineAvailabilityApiV1VaccineAvailabilityGet = (
+  props: ListVaccineAvailabilityApiV1VaccineAvailabilityGetProps,
 ) => (
   <Get<
-    EntriesResponse[],
+    VaccineAvailabilityExpandedResponse[],
     HTTPValidationError,
-    GetEntriesApiV1EntriesGetQueryParams,
+    ListVaccineAvailabilityApiV1VaccineAvailabilityGetQueryParams,
     void
   >
-    path="/api/v1/entries/"
+    path="/api/v1/vaccine-availability"
     {...props}
   />
 );
 
-export type UseGetEntriesApiV1EntriesGetProps = Omit<
+export type UseListVaccineAvailabilityApiV1VaccineAvailabilityGetProps = Omit<
   UseGetProps<
-    EntriesResponse[],
+    VaccineAvailabilityExpandedResponse[],
     HTTPValidationError,
-    GetEntriesApiV1EntriesGetQueryParams,
+    ListVaccineAvailabilityApiV1VaccineAvailabilityGetQueryParams,
     void
   >,
   "path"
 >;
 
 /**
- * Get Entries
+ * List Vaccine Availability
  */
-export const useGetEntriesApiV1EntriesGet = (
-  props: UseGetEntriesApiV1EntriesGetProps,
+export const useListVaccineAvailabilityApiV1VaccineAvailabilityGet = (
+  props: UseListVaccineAvailabilityApiV1VaccineAvailabilityGetProps,
 ) =>
   useGet<
-    EntriesResponse[],
+    VaccineAvailabilityExpandedResponse[],
     HTTPValidationError,
-    GetEntriesApiV1EntriesGetQueryParams,
+    ListVaccineAvailabilityApiV1VaccineAvailabilityGetQueryParams,
     void
-  >("/api/v1/entries/", props);
+  >("/api/v1/vaccine-availability", props);
```

Comment:
Remove the props, as we are not using postal code just yet.
Should be `("/api/v1/vaccine-availability")` for now